content/en/serverless/azure_app_services/azure_app_services_container.md

```diff
@@ -129,65 +130,103 @@ RUN cd /datadog/tracer && tar -zxf datadog-dotnet-apm-2.51.0.tar.gz
 
-To set up logging in your application, see [Node.js Log Collection][3]. To set up trace log correlation, see [Correlating Node.js Logs and Traces][4].
+To set up logging in your application, see [Node.js Log Collection][503]. To set up trace log correlation, see [Correlating Node.js Logs and Traces][504].
 
-[1]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/go
-[2]: /metrics/custom_metrics/dogstatsd_metrics_submission/?code-lang=go#code-examples
-[3]: /logs/log_collection/go/
-[4]: /tracing/other_telemetry/connect_logs_and_traces/go
+[501]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/go
+[502]: /metrics/custom_metrics/dogstatsd_metrics_submission/?code-lang=go#code-examples
+[503]: /logs/log_collection/go/
+[504]: /tracing/other_telemetry/connect_logs_and_traces/go
 {{% /tab %}}
 {{% tab "PHP" %}}
 #### Tracing
-Instrument your main application with the `dd-trace-php` library. See [Tracing PHP applications][1] for instructions.
+Instrument your main application with the `dd-trace-php` library. See [Tracing PHP applications][601] for instructions.
 
 #### Metrics
-Custom metrics are also collected through the tracer. See the [code examples][2].
+Custom metrics are also collected through the tracer. See the [code examples][602].
 
 #### Logs
 The Datadog sidecar uses file tailing to collect logs. Datadog recommends writing application logs to `/home/LogFiles/` because this directory is persisted across restarts. 
 
 You can also create a subdirectory, such as `/home/LogFiles/myapp`, if you want more control over what is sent to Datadog. However, if you do not tail all log files in `/home/LogFiles`, then Azure App Service application logs related to startups and errors are not collected.
 
-To set up logging in your application, see [Node.js Log Collection][3]. To set up trace log correlation, see [Correlating Node.js Logs and Traces][4].
+To set up logging in your application, see [Node.js Log Collection][603]. To set up trace log correlation, see [Correlating Node.js Logs and Traces][604].
 
-[1]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/php/#getting-started
-[2]: /metrics/custom_metrics/dogstatsd_metrics_submission/?code-lang=php#code-examples
-[3]: /logs/log_collection/php/
-[4]: /tracing/other_telemetry/connect_logs_and_traces/php
+[601]: /tracing/trace_collection/automatic_instrumentation/dd_libraries/php/#getting-started
+[602]: /metrics/custom_metrics/dogstatsd_metrics_submission/?code-lang=php#code-examples
+[603]: /logs/log_collection/php/
+[604]: /tracing/other_telemetry/connect_logs_and_traces/php
 {{% /tab %}}
 {{< /tabs >}}
 
-### Sidecar container
+### Instrumentation
+
+Instrumentation is done through a sidecar container that runs alongside your main application container. This sidecar container collects traces, metrics, and logs from your main application container and sends them to Datadog.
```

Comment:
```suggestion
Instrumentation is done using a sidecar container. This sidecar container collects traces, metrics, and logs from your main application container and sends them to Datadog.
```

Reply:
since it's a sidecar, I suggest that "that runs alongside your main application container" is unnecessary.